src/biconnectivity/biconnect.jl

```diff
@@ -31,14 +31,13 @@ function biconnected_components(g::Graph)
             empty!(state.stack)
         end
     end
-
     return state.biconnected_comps
 end
 
 """
 Does a DFS visit and stores the depth and low-points of each vertex
 """
-function visit!(g::Graph, state::Biconnections, u::Int, v::Int)
+function visit!(g::Graph, state::Biconnections, u::Integer, v::Integer)
```

Comment:
Yes - I think we should abstract this. We need to be prepared to troubleshoot `MethodError`s on the chained functions that eventually enforce the traits, though. Does that work?